service/storage_service.hh

```diff
@@ -447,9 +447,12 @@ public:
         daemon.deactivate();
     }
 #endif
-public:
-    future<std::unordered_set<token>> prepare_replacement_info(const std::unordered_map<gms::inet_address, sstring>& loaded_peer_features);
+private:
+    // Tokens and streams of the replaced node.
+    using replacement_info = std::pair<std::unordered_set<token>, db_clock::time_point>;
+    future<replacement_info> prepare_replacement_info(const std::unordered_map<gms::inet_address, sstring>& loaded_peer_features);
```

Comment:
This patch changes code in storage_service.cc. The subject starts with cdc, while the previous 2 starts with storage_service. Let's make it consistent.

Reply:
Hopefully I made the commit log consistent.